### ISSUE TITLE
[ENH]: Provide hook to initialize ToMany Instances

### ISSUE THREAD

**Issue by GoogleCodeExporter** [magritte-metamodel/magritte#18]
```
** Hold until requested. I needed this, but worked around it for the time 
being. If it comes up again, here's the result of my initial investigation...

The idea is to allow model constraints (e.g. between the parent and the new 
reference) to be set up before the new object is viewed.

Implementation notes:
1. The To Many Morph/Component would send "description initialize: self 
selected" instead of "self selected new"
2. MAToManyRelationDescription>>#initialize: aBlock (taking a class as an 
argument

Would be used like:

    MAToManyRelationDescription new
        accessor: #events;
        label: 'Events';
    classes: { MyDomainClass AnotherClass };
    initialize: [ :klass | klass new date: self date ].
```

Original issue reported on code.google.com by `sean.p.d...@gmail.com` on 9 Oct 2013 at 4:40

**Comment by seandenigris** [comment]
Magritte-Model-SeanDeNigris.463

[WIP]: Finally give it a try. Usage example setting relation data specific to current object:
```
description initializer: [ :e | 
	e
		date: self date;
		payee: self vendor ]
```

**Comment by seandenigris** [comment]
Seems to be resolved by 6b1ce04
